### DISCUSSION TITLE
Docker-compose: "DB migration" reset DB after docker-compose down

### DISCUSSION THREAD

**Discussion by Franc067** [WeblateOrg/weblate#6845] [Q&A] [answered]
Hello,
I have installed weblate in a docker-compose environment. 
Weblate is running fine, it can communicate with the postgres and  redis dicker instances and we can create users and other stuffs that I can find in the DB.

When I run a "docker -compose down"  everything stop correctly, but, then, when I start the infra with a docker-compose up  -d" the DB is reset during the "database migration" job

here some logs

```
Starting...
Postgres is up
Starting database migration...
Operations to perform:
  Apply all migrations: accounts, addons, admin, auth, authtoken, checks, configuration, contenttypes, fonts, gitexport, glossary, lang, memory, metrics, screenshots, sessions, social_django, trans, utils, weblate_auth, wladmin
Running migrations:
  Applying weblate_auth.0001_initial... OK
  Applying lang.0001_squashed_0008_auto_20200408_0436... OK
  Applying trans.0001_squashed_0074_fix_broken_browser_alert... OK
  Applying trans.0075_auto_20200501_1428... OK
  Applying trans.0076_comment_userdetails... OK
  Applying trans.0077_auto_20200513_1512... OK
  Applying trans.0078_auto_20200515_0729... OK
```

As someone already had this issue and how I can avoid this behaviour
Thank you for your help

**Comment by nijel** [comment, marked as answer]
Maybe you don't have a persistent volume for the PostgtreSQL container so that it always starts with a blank database?

Check https://docs.weblate.org/en/latest/admin/install/docker.html#installation and https://github.com/WeblateOrg/docker-compose/blob/main/docker-compose.yml

**Comment by Franc067** [comment]
Hello, thank you that's exactly the point, 
Thank you for your help and sorry to take your time
